Sources/core/objects/Human.cpp

```diff
@@ -125,7 +125,22 @@ void Human::processGUImsg(const Message2 &msg)
     if (msg.type == MessageType::MESSAGE)
     {
         std::string text = obj["text"].toString().toStdString();
-        GetGame().GetChat().PostWords(name, text, owner.Id());
+        std::string prefixes[] = {"me", "me ", "*", "* "};
```

Comment:
Now your code will never match "me " (and "* "), because firstly it will match just "me" and after that will change "me something" into "Morgan James  something" (two spaces). In order to avoid that you need to reorder prefixes - "me " should be before "me", and "* " should be before "*".